server/config/contactForm.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  CONCERT_MAIL: ['concert@kfet-insa.fr', 'technique@kfet-insa.fr'],
+  EVENT_MAIL: ['sg@kfet-insa.fr', 'president@kfet-insa.fr'],
+  LOST_MAIL: ['sg@kfet-insa.fr'],
+  WEBSITE_MAIL: ['webmaster@kfet-insa.fr'],
+  RECAPTCHA: {
+    SECRET: process.env.RECAPTCHA_SECRET || '6LfIS1gUAAAAAKFZgZm1ZOVhzEFDMrD8GcV2W2X2',
+  },
```

Comment:
We should seek for a nice and configurable way to choose mails:
Maybe just use an env variable with a separator `,`, It will allow to not have any configuration (with a dns or key or anything else) on git

Reply:
Ok, so let's do:
- CONTACT_CONCERT_MAIL = `concert@kfet-insa.fr,technique@kfet-insa.fr,sg@kfet-insa.fr`
- CONTACT_EVENT_MAIL = `sg@kfet-insa.fr,president@kfet-insa.fr`
- CONTACT_LOST_MAIL = `sg@kfet-insa.fr`
- CONTACT_WEBSITE_MAIL = `webmaster@kfet-insa.fr`

For dev env, we will send mails to `contact@fake.com` hardcoded

Ok ?